### ISSUE TITLE
Ensure area-hidden elements are no focusable nor contain focusable elements

### ISSUE THREAD

**Issue by Xiaoyu-Huang** [Azure/LogicAppsUX#4698]
### Describe the issue

Run Accessibility insights extension fast pass, it detected issues on info icon on action panel. Please see image below.

![image](https://github.com/Azure/LogicAppsUX/assets/22843713/47220bcb-e269-490c-b6bb-d035ef7066fd)

Title: WCAG 4.1.2: Ensures aria-hidden elements are not focusable nor contain focusable elements (.msla-designer-tools)
Tags: Accessibility, WCAG 4.1.2, aria-hidden-focus

Issue: Ensures aria-hidden elements are not focusable nor contain focusable elements (aria-hidden-focus - https://accessibilityinsights.io/info-examples/web/aria-hidden-focus)

Target application: IcM - https://ppeportal.microsofticm.com/imp/v5/automation/workflows/create

Element path: .msla-designer-tools

Snippet: <div class="msla-designer-tools" aria-hidden="true">

How to fix: 
Fix all of the following:
  Focusable content should be disabled or be removed from the DOM

Environment: Microsoft Edge version 124.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.41.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

### Are you using new designer or old designer

New Designer (With action panels)

### Browser

Browser: Edge

logic app library version:

"@microsoft/logic-apps-designer": "4.1.1",
"@microsoft/logic-apps-shared": "4.1.1"

**Comment by hartra344** [comment]
Not sure how this one was getting flagged, its completely focusable and nothing around it is area-hidden

https://github.com/Azure/LogicAppsUX/assets/13208452/f722c9d3-0243-45f5-a3ec-16b8c67b6e99
